.github/workflows/doc-bot.yml

```diff
@@ -12,7 +10,7 @@ jobs:
     runs-on: ubuntu-22.04
     env:
       CI_COMMIT_MESSAGE: Generated docs for ${{ github.sha }}
-      GH_TOKEN: ${{ github.token }}
+      GH_TOKEN: ${{ secrets.GH_ACTION_ACCESS_TOKEN }}
```

Comment:
do we need this if we have set the token below?

Reply:
good point, probably not!

Reply:
I re-read the docs, don't think we need the one below and just need to set the env variable